### ISSUE TITLE
Emission Factor Models for other countries

### ISSUE THREAD

**Issue by micaelbatista** [datagir/nosgestesclimat#1731]
Hi,

I'd like to add a model for Portugal in your tool.
Is your tool ready to add models for other countries besides France?
If the tool is ready, what do I need to prepare in my end?

**Comment by Clemog** [comment]
Hi @micaelbatista ! Thanks for your commitment in Nos Gestes Climat ! [Here is a first version of a tutorial](https://github.com/datagir/nosgestesclimat/blob/master/INTERNATIONAL.md) (**in French for now, we are very sorry about that**, you are one of the first non-francophone contributor).

The idea is to create a file like https://github.com/datagir/nosgestesclimat/blob/master/data/i18n/models/CH.yaml or https://github.com/datagir/nosgestesclimat/blob/master/data/i18n/models/CH-en-us.yaml for Portugal, containing rules from French model to be overwritten. The aim is to keep base rules common to all countries (improvements from ones model may be useful for others) but custom attributes for default values, emission factors etc..

**Comment by Clemog** [comment]
Switzerland version of flight model is discussed in #1725 for instance.

**Comment by Clemog** [comment]
> So I can override all the items with a constant `formule` or `par défaut`.
> The items that have formulas in the `formule` or `par défaut` fields, shouldn't be touched.
> After I finish all the overrides, I should dump them all in a `data/i18n/models/PT.yaml` file.
> Is this correct?

Yes, this is the idea. You can even dump it into `PT-en-us.yaml`, https://github.com/datagir/nosgestesclimat/pull/1747 will help to translate and putit into `PT-fr.yaml`. However, do not forget to add regions params in your file (for now, we assume that i has to be in the -fr version.

```
params:
  nom: Portugal
  code: PT
  gentilé: portugaise
```

**Comment by Clemog** [comment]
> Also, each override in the file, should have the `titre`, `description`, `note` and `référence` fields?
> What is the purpose of each of these fields?

Sorry there is no translations for our mechanisms. My advice is to add a `titre`, `description`, `note` if you want attributes from base rules to be overwritten. `titre` is a title for your rule (useful to make it clear in [docs](https://nosgestesclimat.fr/documentation)), `description`, for questions, corresponds to the help button ℹ️, `note` corresponds to explanation you want to add to a specific rule to explain your calculation, `references` should contain your sources

**Comment by Clemog** [comment]
> Finally, should I override personas as well, or not yet?

For now, we do not enable specific personas for regions but you can create it for the base model as the rules shouldn't be different between regions for now : rules structure is assumed to be the same for all regions for now.

**Comment by micaelbatista** [comment]
Hi everyone!

I've been having some trouble inputting the files into the repo, could you help me put the files in the right place and check if there're correct?
This is a portuguese translation done by hand with a translation here in our portuguese non-profit Transitar.
There might be translations missing, due to the time it took to translate all of it, but we can translate whatever is missing.

Here are the translated files: https://drive.google.com/drive/folders/12WORuZ9Hr9Mbxb19kAEIYMgohioCyjQr?usp=sharing
